R/calcEmissions4ReportExtra.R

```diff
@@ -0,0 +1,18 @@
+#' Calculates historical emissions needed for estimating emission factors
+#' in extra emissions reporting
+#'
+#' @param source either "PBL_2007" or "PBL_2022"
+#'
+#'
+#' @return list of magclass with REMIND input data for
+#' different sectors for timesteps 2000-2100, in Mt N per year
+#' @author Gabriel Abrhahao
+# nolint start
```

Comment:
Don't activate linters. In mrremind they will not make buildLibrary() fail anyways